### ISSUE TITLE
[Go] found a broken connection logs

### ISSUE THREAD

**Issue by leszko** [hazelcast-guides/kubernetes-external-client#16]
I tried the Go client and it seems to work, but prints from time to time the following log:
```
2021/09/13 16:41:41 INFO : found a broken connection to: 10.164.0.3:5701, trying to fix it.
```
@yuce could you please check?

**Comment by yuce** [comment]
Here is the relevant ticket: https://github.com/hazelcast/hazelcast-go-client/issues/600 This issue is fixed at: https://github.com/hazelcast/hazelcast-go-client/pull/617 and it is merged.

**Comment by yuce** [comment]
We've released Go client v1.1.0 on Tuesday. Did that fix the issue?

**Comment by leszko** [comment]
To be honest, now I wasn't able to reproduce it even with v1.0.0... closing.
